app/src/main/java/com/example/teamyuml/decentworkmobile/AppSingleton.java

```diff
@@ -0,0 +1,43 @@
+package com.example.teamyuml.decentworkmobile;
+
+import android.content.Context;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.toolbox.Volley;
+
+/*
+ * This class is responsible
+ * for configuration of Volley package
+ */
+public class AppSingleton {
+    private static AppSingleton mAppSingletonInstance;
```

Comment:
Add docstring to class fields pls cuz @KamilSupera won't know what these fields doing.